rmw_connext_cpp/src/functions.cpp

```diff
@@ -31,25 +31,25 @@
 #  pragma clang diagnostic ignored "-Wreturn-type-c-linkage"
 # endif
 #endif
-#include <ndds/ndds_cpp.h>
-#include <ndds/ndds_requestreply_cpp.h>
+#include "ndds/ndds_cpp.h"
+#include "ndds/ndds_requestreply_cpp.h"
```

Comment:
I don't think these count as our headers, and so I believe they should be included with `<>` and should be treated something like boost would be. I think they should fall under "Other libraries' .h files." as described in https://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Names_and_Order_of_Includes. To be fair, the guide doesn't clearly state which include style should be used for "Other libraries' .h files.".

This might be a case which isn't covered by the linter and in which case we need to clearly state what we do and make the linter agree with a patch. In my opinion it should be something like this example:

(`project/src/foo.cpp`)
```c++
#include "project/foo.hpp"

#include <stdio.h>

#include <iostream>

#include <boost/thread.hpp>

#include "project/other.hpp"
```

Thoughts?